.github/workflows/check-patch.yaml

```diff
@@ -14,6 +14,8 @@ jobs:
       with:
         python-version: 3.8
     - name: Install requirements
-      run: bash ./scripts/ci/install.sh
-    - name: Check Patch
-      run: ./scripts/ci/check_patch.sh
+      run: pip install ".[all,tests]" Pygments collective.checkdocs pre-commit
+    - name: Check README
+      run: python setup.py checkdocs
+    - name: Check formatting
+      run: pre-commit run --all-files
```

Comment:
@pared Btw, would it make sense to include this into the test workflow as a separate job? Just a bit weird that this is a separate one, because it is just a yet another test.

Reply:
Also, have you considered `fail-fast: false` for the test build matrix? Seems strange that all jobs are getting cancelled because of one flaky one.

Reply:
@efiop yeah, I guess it makes sense. In `dvc-bench` we have style checks run before test. If we want to restrict check to particular os-es, we can do that by using `if` statement. Also `fail-fast` sounds like a good idea.